.travis.yml

```diff
@@ -1,8 +1,12 @@
 language: node_js
 node_js: node
 os:
-  - osx
+- osx
```

Comment:
OSX still required?

Reply:
Nah - I am just experimenting to get the screenshots.  I plan to remove this.  :)